### DISCUSSION TITLE
AttributeError: 'ConcurrencyLimiter' object has no attribute 'set_search_properties'

### DISCUSSION THREAD

**Discussion by hanhanwu** [microsoft/FLAML#152] [Q&A]
I tried multiple dataset but always got error saying, "AttributeError: 'ConcurrencyLimiter' object has no attribute 'set_search_properties'"

Then I tried the simple 2 examples FLAML provided here: https://github.com/microsoft/FLAML, the iris and boston examples, same error. 
![WTF](https://user-images.githubusercontent.com/4024769/128588193-2bc5169e-b570-4fd4-9e64-2b8554fe46b1.PNG)

I have also tried some settings related to ConcurrencyLimiter, none worked.

Do you know how to fix this issue?

**Comment by sonichi** [comment]
Hi @hanhanwu were you using a ray version <1.0.0? We'll do a version check of ray in the next version to avoid this error. Thanks.

**Reply by hanhanwu** [reply]
Oh this one has already been solved in https://github.com/microsoft/FLAML/issues/153, I mistakenly created 2 posts... Please ignore this post, thank you!
